bHY6yh-de3258/cod-vyg67d_bHY6yh-de3258

```diff
@@ -0,0 +1,233 @@
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="V" n="cod-pu6y7d">Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-13" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#O" n="14-r"/>
+            <cb ed="#O" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de3258"><!-- p1l1-r12 -->
+            <head xml:id="bHY6yh-de3258-Hd1e108">Rubrica 12</head>
+            <head xml:id="bHY6yh-de3258-Hd1e111" type="question-title">De timore Dei</head>
+            <p xml:id="bHY6yh-de3258-d1e114">
+               De ti<lb ed="#O" break="no"/>more 
+               dei. Rubrica. 12
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e119">
+               <lb ed="#O"/><ref xml:id="bHY6yh-de3258-Rd1e116">Iob hystoria capitulo 28.</ref> <quote xml:id="bHY6yh-de3258-Qd1e123">Timor domi<lb ed="#O" break="no"/>ni 
+               ipse est sapientia.</quote> 
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e129">
+               <lb ed="#O"/><ref xml:id="bHY6yh-de3258-Rd1e128">Moyses in deuteronomio. capitulo 6.</ref> <quote xml:id="bHY6yh-de3258-Qd1e133">ti<lb ed="#O" break="no"/>meas 
+               dominum deum tuum: et custodi
+               <lb ed="#O"/>as mandata et precepta eius. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e142">
+               <lb ed="#O"/><ref xml:id="bHY6yh-de3258-Rd1e144">Psalmista. 2 psalmus.</ref> ¶ <quote xml:id="bHY6yh-de3258-Qd1e146">Seruite domino 
+               in timore et exultate ei cum 
+               <lb ed="#O"/>tremore. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e152">
+               ¶ <ref xml:id="bHY6yh-de3258-Rd1e154">28. psalmus.</ref> <quote xml:id="bHY6yh-de3258-Qd1e154">firmamentum est 
+               <lb ed="#O"/>dominus timentibus eum. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e160">
+               [omitted]
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e168">
+               ¶ <ref xml:id="bHY6yh-de3258-Rd1e173">84. psalmus.</ref> <quote xml:id="bHY6yh-de3258-Qd1e170">prope 
+               <lb ed="#O"/>timentes [omitted] salutare ipsius. </quote>
```

Comment:
don't write omitted for individual missing words, but only for whole paragraphs that are missing. (For example what you did on lines 80-82 is perfect.

But we don't need an omitted statement here.

```suggestion
               <lb ed="#O"/>timentes salutare ipsius. </quote>
```